input/pagecontent/export.md

```diff
@@ -601,12 +625,16 @@ Note that if a server copies files to the Bulk Data output endpoint or proxies r
 
 ### Server Capability Documentation
 
-This implementation guide is structured to support a wide variety of Bulk Data Export use cases and server architectures. To provide clarity to developers on which capabilities are implemented in a particular server, server providers SHOULD ensure that their Capability Statement accurately reflects the implemented Bulk Data Operations and that their documentation addresses the topics below. Future versions of this IG may define a computable format for this information as well.
+This implementation guide is structured to support a wide variety of Bulk Data Export use cases and server architectures. To provide clarity to developers on which capabilities are implemented in a particular server, server providers SHALL ensure that their Capability Statement accurately reflects the implemented Bulk Data Operations. Additionally, the server's Capability Statement SHOULD list the resource types available for export in the `rest.resource` element, and SHOULD list the search parameters that can be used in the _typeFilter parameter in the `rest.resource.searchParam` element. 
+
+Servers SHOULD indicate resource types and search parameters that are accessible on the server with the REST API, but not available using the Bulk Export operation, with one or more extensions that have a URL of `http://hl7.org/fhir/uv/bulkdata/Extension/bulk-export-not-supported` and a `valueCode` of `any-level`, `patient-level`, `group-level` or `system-level` to indicate which bulk operations are not supported.
```

Comment:
Will you add this extension definition in this PR?

Reply:
Yep, was just waiting to see if we had agreement on the idea of the extension. Another thought - instead of using a valueCode, what do you think about using a valueCanonical for the operations that aren't supported? This would avoid the need for an additional code system and could potentially generalize to use cases outside of bulk as well?